yang/ietf-network-inventory.yang

```diff
@@ -232,6 +233,13 @@ module ietf-network-inventory {
         "The alias name of the entity (e.g., component). This alias
          name can be specified by network manager.";
     }
+  }
+
+  grouping ne-component-common-entity-attributes {
+    description
+      "The set of attributes which are common to all the entities
+       (e.g., component, network elements) defined in this module.";
+ 
```

Comment:
I would define this as a super-set of the common attributes which includes also the basic common attributes:

```suggestion
 
 uses basic-common-entity-attributes;
 
```

Reply:
In the description we talk about "splitting" that is referring to have have 2 sub-sets of the original grouping, but nothing prevent to change our mind and create just a sub-set as required by Bo and maintain the original common grouping as superset . I think the important point is to agree on the number of attributes that would be part of the "minimal" (or "basic") grouping. For that I would wait for Bo opinion and review.